LockwiseXCUITests/LockwiseScreenGraph.swift

```diff
@@ -268,7 +269,9 @@ extension BaseTestCase {
         settings.cells.staticTexts["Passwords & Accounts"].tap()
         settings.cells.staticTexts["AutoFill Passwords"].tap()
         waitforExistence(settings.switches["AutoFill Passwords"], timeout: 3)
-        settings.switches["AutoFill Passwords"].tap()
+        if let switchValue = settings.switches["AutoFill Passwords"].value as? Bool, switchValue == false {
```

Comment:
2 nits: 

1. put the second clause on a new line
2. use `not switchValue`, or the equivalent. `switchValue` is already a `Bool`, so we don't need to compare with a value.